apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as fsx from 'fs-extra';
+import * as os from 'os';
+import * as colors from 'colors';
+
+import {
+  ICustomOption,
+  ICustomEnumOption,
+  ICustomEnumValue,
+  Stopwatch,
+  Event
+} from '../../index';
+
+import {
+  CommandLineFlagParameter,
+  CommandLineIntegerParameter,
+  CommandLineStringListParameter,
+  CommandLineOptionParameter,
+  ICommandLineActionOptions
+} from '@microsoft/ts-command-line';
+
+import RushCommandLineParser from './RushCommandLineParser';
+import { BaseRushAction } from './BaseRushAction';
+import { TaskManager } from '../utilities/TaskManager';
+
+interface ICustomOptionInstance extends ICustomOption {
```

Comment:
>ICustomOptionInstance [](start = 10, length = 21)

Why do we need this?

Reply:
It stores the parameter value

---
In reply to: [146732192](https://github.com/Microsoft/web-build-tools/pull/387#discussion_r146732192) [](ancestors = 146732192)